engines/ags/engine/gfx/gfx_util.cpp

```diff
@@ -65,7 +65,7 @@ static const BlendModeSetter BlendModeSets[kNumBlendModes] = {
 };
 
 bool SetBlender(BlendMode blend_mode, bool dst_has_alpha, bool src_has_alpha, int blend_alpha) {
-	if (blend_mode < 0 || blend_mode > kNumBlendModes)
+	if (blend_mode < 0 || blend_mode >= kNumBlendModes)
```

Comment:
This should probably be fixed upstream, in AGS' repo:
https://github.com/adventuregamestudio/ags/blob/9bb443b18e5b3dc742c0d6a5c7207358c6ad39f2/Engine/gfx/gfx_util.cpp#L69C40-L69C54